apps/src/templates/ChallengeDialog.jsx

```diff
@@ -165,7 +166,7 @@ const ChallengeDialog = Radium(React.createClass({
         </div>}
       </BaseDialog>
     );
-  },
-}));
+  }
+}
 
-export default ChallengeDialog;
+export default Radium(ChallengeDialog);
```

Comment:
Not specific to this change: do we need `Radium` here?  I'm not seeing any pseudo classes in the styles, or arrays of styles.

Reply:
Great question. The component used Radium before the conversion so I kept it during ES6ification, but I can look into why/if we needed it in the first place.